### DISCUSSION TITLE
Is it possible to use firebase-js-sdk in React native

### DISCUSSION THREAD

**Discussion by DanijelL21** [firebase/firebase-js-sdk#8669] [Q&A]
Hi all,

I'm using  firebase-js-sdk  like

```
import { initializeApp } from "firebase/app";
import { getDatabase, ref, get, child } from "firebase/database";
import { firebaseConfig } from "../secrets/firebaseConfig";

// Initialize the app and database
const app = initializeApp(firebaseConfig);

const database = getDatabase(app);
```

and it's working for me in React native (my app is deployed on App store). Now I want to use app check but I don't know if that is possible within firebase-js-sdk  on React native. Can someone please help. Thank you very much

**Comment by prathmesh235** [comment]
The Firebase JavaScript SDK's App Check support is documented for web apps, with providers such as reCAPTCHA Enterprise.

For a React Native app, I would recommend checking the Firebase React Native/mobile SDK support for App Check rather than relying on the web `firebase/app-check` package directly.

If you're using React Native on iOS or Android, the native App Check providers (App Attest/DeviceCheck on iOS and Play Integrity on Android) are the relevant approach.
